airbyte-integrations/connectors/source-shopify/source_shopify/source.py

```diff
@@ -53,7 +60,7 @@ def request_params(self, next_page_token: Mapping[str, Any] = None, **kwargs) ->
             params.update(**next_page_token)
         else:
             params["order"] = f"{self.order_field} asc"
-            params[self.filter_field] = self.config["start_date"]
+            params[self.filter_field] = self.default_filter_field_value
```

Comment:
I think we should remove the `order` parameter when using `since_id`, because it's not allowed to use both. 
The output of a request with `order` and `since_id` paramters:
```json
{
	"errors": {
		"order": "order cannot be passed when since_id is present."
	}
}
```